packages/react/src/components/molecules/ResultsHeading/index.js

```diff
@@ -17,21 +17,29 @@ import ButtonToggle from 'MayflowerReactButtons/ButtonToggle';
 import SelectBox from 'MayflowerReactForms/SelectBox';
 import Tags from 'MayflowerReactMolecules/Tags';
 
+function getTitle(resultsHeading) {
+  if (resultsHeading.title !== undefined) {
```

Comment:
```suggestion
  if (resultsHeading.title) {
```
I think it should check if it has a truthy value here